deb/publish.go

```diff
@@ -452,6 +454,36 @@ func (p *PublishedRepo) GetLabel() string {
 	return p.Label
 }
 
+// GetAppStream returns full path list of appstream files
+func (p *PublishedRepo) GetAppStream(component string) ([]string, error) {
+	if utils.Config.AppStreamDir == "" {
```

Comment:
The way `aptly` is implemented, it is supposed that `Config` object is not accessed from this level.

I would suggest two changes:

* push `AppStreamDir` to `Context`
* pass `AppStreamDir` to this method as parameter (same below in `publish.go`).